.vuepress/config-path-checker/utils.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  isArray: function (element) {
+    if (Array.isArray(element)) return true
+    return false
+  },
```

Comment:
Why did you not use directly `Array.isArray()`? I understand the syntax sugar for the `isObject`, but less for `isArray`

Reply:
I think I was annoyed by the length of `Array.isArray()`. But yes, it's unnecessary complexity.